src/dataflow/core/lispress.py

```diff
@@ -591,16 +590,6 @@ def unnest_line(
             result_exprs.append(lambda_expr)
             return result_exprs, idx, idx, var_id_bindings
 
-        elif hd == SEQUENCE:
-            # handle programs that have multiple statements sequenced together
-            result_exprs = []
-            arg_idx = idx  # in case `tl` is empty
-            for statement in tl:
-                exprs, arg_idx, idx, var_id_bindings = unnest_line(
-                    statement, idx, var_id_bindings
-                )
-                result_exprs.extend(exprs)
-            return result_exprs, arg_idx, idx, var_id_bindings
```

Comment:
`do` is no longer special-cased, so it now appears in the program just like any other function.